src/adaptors.h

```diff
@@ -134,11 +134,30 @@ template <typename Vector> inline auto makeVectorFromJsNumberArray(v8::Local<v8:
 
     if (!num->IsNumber())
       throw std::runtime_error{"Expected array element of types Number"};
-
+      //std::cout<<"hello"<<std::endl;
     vec.at(atIdx) = Nan::To<std::int32_t>(num).FromJust();
   }
 
   return vec;
 }
 
+// our function for changing js array into  vector of long long int
+
+template <typename Vector> inline auto makeVectorFromJsNumberArray1(v8::Local<v8::Array> array) {
```

Comment:
sir , i needed to make a vector of  std::int64_t  thats why i wrote a different method .